### ISSUE TITLE
Gaussian blur that's a multiple of 10 will skew the image.

### ISSUE THREAD

**Issue by jawngee** [gdawg/uiimage-dsp#3]
Any blur that's a multiple of 10 skews the image 45 degrees.

**Comment by gdawg** [comment]
weird, do you know if it's only multiples of 10? I'll have a look and see if I can figure out what's going on.

**Comment by gdawg** [comment]
whoops - accidently closed instead of just commenting.

**Comment by jawngee** [comment]
Yeah it's any multiple of 10.  I'm not a math guy, but my only guess is the kernel size?

**Comment by gdawg** [comment]
still haven't had a chance to look at this, been super busy and am on holidays for the next month, perhaps in nov ;(

**Comment by ThaiSeb** [comment]
Well, it seems it also happen with other radius values that are not odd. Based on some testing with different image sizes, the images can get skewed if the radius is not odd.

**Comment by gdawg** [comment]
can either of you provide a radius which will cause the example to skew on an actual device? 

I've done some testing this morning and can't find any radius which causes skewing on a phone. The simulator skews with radius 10 but I believe this to be a bug in the simulator code. 

You can easily try different radius by modifying the switch statement for the "advanced" section in the example
